services/accounts/accounts.go

```diff
@@ -29,3 +30,12 @@ func (api *API) SaveAccounts(ctx context.Context, accounts []accounts.Account) e
 func (api *API) GetAccounts(ctx context.Context) ([]accounts.Account, error) {
 	return api.db.GetAccounts()
 }
+
+func (api *API) DeleteAccount(ctx context.Context, address common.Address) error {
+	err := api.db.DeleteAccount(address)
+	if err != nil {
+		return err
+	}
+	api.feed.Send(address)
```

Comment:
Do we need this? `api.feed` takes various arguments, not sure what it is useful for.